index.html

```diff
@@ -183,12 +175,11 @@ <h1>Make your own comic</h1>
   <script src="https://unpkg.com/bootstrap@4.3.1/dist/js/bootstrap.bundle.min.js"></script>
   <script src="https://unpkg.com/lodash@4.17.11/lodash.min.js"></script>
   <script src="https://unpkg.com/g1@0.17.0/dist/g1.min.js"></script>
-  <script src="https://unpkg.com/showdown@1.9.0/dist/showdown.min.js"></script>
-  <script src="https://unpkg.com/highlightjs@9.12.0/highlight.pack.min.js"></script>
-  <script src="https://unpkg.com/clipboard@2.0.4/dist/clipboard.min.js"></script>
-  <script src="https://unpkg.com/plain-draggable@2.5.12/plain-draggable.min.js"></script>
-  <script src="https://unpkg.com/save-svg-as-png@1.4.14/lib/saveSvgAsPng.js"></script>
-  <script src="dist/comicgen.min.js"></script>
-  <script src="docs/script.js"></script>
+  <script src="https://unpkg.com/showdown@1.9.0/dist/showdown.min.js" defer></script>
+  <script src="https://unpkg.com/highlightjs@9.12.0/highlight.pack.min.js" defer></script>
+  <script src="https://unpkg.com/clipboard@2.0.4/dist/clipboard.min.js" defer></script>
+  <script src="dist/comicgen.min.js" defer></script>
+  <script src="node_modules/fabric/dist/fabric.js" defer></script>
+  <script src="docs/script.js" defer></script>
```

Comment:
Could you please send the `defer` as a separate PR? I'd like to independently evaluate the effectiveness of this.